sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -1990,61 +1990,144 @@ test_suite(
+            "offset": SLOTS["b"],
         },
     },
     devices = ["fpga_cw310"],
-) for slot, offset in SLOTS.items()]
+)
+
+[otp_image(
+    name = "otp_img_sigverify_always_{}".format(lc_state.lower()),
+    src = "//hw/ip/otp_ctrl/data:otp_json_{}".format(lc_state.lower()),
+    overlays = STD_OTP_OVERLAYS,
+) for lc_state in structs.to_dict(CONST.LCV)]
+
+# Splice OTP images into bitstreams
+[
+    bitstream_splice(
+        name = "bitstream_sigverify_always_{}".format(lc_state.lower()),
+        src = "//hw/bitstream:rom",
+        data = ":otp_img_sigverify_always_{}".format(lc_state.lower()),
+        meminfo = "//hw/bitstream:otp_mmi",
+        tags = ["vivado"],
+        update_usr_access = True,
+    )
+    for lc_state in structs.to_dict(CONST.LCV)
+]
 
 SIGVERIFY_BAD_CASES = [
     {
-        "desc": "bad",
-        "suffix": "_corrupted",
+        "a": "nothing",
+        "b": "bad",
+        "expected_bfv": hex(CONST.BFV.SIGVERIFY.BAD_ENCODED_MSG),
+    },
+    {
+        "a": "bad",
+        "b": "nothing",
+        "expected_bfv": hex(CONST.BFV.SIGVERIFY.BAD_ENCODED_MSG),
+    },
+    {
+        "a": "bad",
+        "b": "bad",
+        "expected_bfv": hex(CONST.BFV.SIGVERIFY.BAD_ENCODED_MSG),
     },
     {
-        "desc": "empty",
-        "suffix": "_empty",
+        "a": "nothing",
+        "b": "nothing",
+        "expected_bfv": hex(CONST.BFV.BOOT_POLICY.BAD_IDENTIFIER),
     },
 ]
 
 [
     opentitan_functest(
-        name = "sigverify_always_a_{}_b_{}".format(
-            a["desc"],
-            b["desc"],
+        name = "sigverify_always_a_{}_b_{}_{}".format(
+            case["a"],
+            case["b"],
+            lc_state.lower(),
         ),
         cw310 = cw310_params(
+            bitstream = ":bitstream_sigverify_always_{}".format(lc_state.lower()),
             exit_failure = MSG_PASS,
-            exit_success = DEFAULT_TEST_FAILURE_MSG,
+            exit_success = MSG_TEMPLATE_BFV_LCV.format(
+                case["expected_bfv"],
+                hex(lc_state_val),
+            ),
+            otp = ":otp_img_sigverify_always_{}".format(lc_state.lower()),
+            tags = ["vivado"] + maybe_skip_in_ci(lc_state_val),
         ),
         key = "multislot",
-        ot_flash_binary = ":boot_policy_valid_img_a_{}_b_{}".format(
-            a["desc"],
-            b["desc"] if a != b and b != "empty" else "bad",
+        ot_flash_binary = ":sigverify_always_img_a_{}_b_{}".format(
+            case["a"],
+            case["b"],
         ),
         targets = ["cw310_rom"],
     )
-    for a in SIGVERIFY_BAD_CASES
-    for b in SIGVERIFY_BAD_CASES
+    for case in SIGVERIFY_BAD_CASES
+    for lc_state, lc_state_val in structs.to_dict(CONST.LCV).items()
 ]
 
 test_suite(
     name = "sigverify_always",
+    tags = ["manual"],
     tests = [
-        "sigverify_always_a_{}_b_{}".format(
-            a["desc"],
-            b["desc"],
+        "sigverify_always_a_{}_b_{}_{}".format(
```

Comment:
nit: name needs to be updated since we changed it above.

Reply:
fixed.